source/get-started/setup/index.md

```diff
@@ -0,0 +1,203 @@
+          <li>Launch a new Finder window</li>
+          <li>Navigate to where you want to project to live</li>
+          <li>File -> New Folder (or Ctrl + Shift + N)</li>
+          <li>Name the folder</li>
+        </ul>
+      </div>
+      <div class="col-xs-12 col-sm-12 col-md-6">
+        <h3>Using Command Line</h3>
+        <p>
+          <ul>
+            <li>Open Terminal</li>
+            <li>Navigate to where you want the folder<br />
+              <kbd>
+                cd "path_to/projects"
+              </kbd>
+            </li>
+            <li>Create the directory with the project name<br/>
+              <kbd>
+                mkdir myProject
+              </kbd>
+            </li>
+          </ul>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>Use Bower To Install PatternFly</h2>
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12">
+        <div class="well">
+          Bower is a package manager. We are going to use Bower to install the PatternFly framework.
+        </div>
+        <p>The first thing needs to be done is to install Bower globally. This is done using the command <kbd>npm install -g bower</kbd></p>
+        <p>Occasionally, you will have to run this command as an Administrator. To do so, precede <kbd>npm install -g bower</kbd> with <kbd>sudo</kbd> and run the command again.</p>
+        <p>Verify that you are in the correct project folder before installing PatternFly. Once you have done so, run <kbd>bower install patternfly</kbd></p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>File Structure</h2>
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12">
+        <p>Your folder should now look like this:</p>
+        <div class="col-sm-8 col-md-8 section-img">
+          <img src="{{ site.baseurl}}assets/img/get-started-file-structure.jpg" alt="">
+        </div>
+        <div class="col-sm-4 col-md-4">
+          <b>TIP</b> Remember never to change the content of the /bower_components folder. Doing so will cause issues with the PatternFly update process.
+        <div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>Add Styles</h2>
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12">
+        <div class="well col-md-12">
+          <b>HEADS UP</b> PatternFly is based off of <a href="https://www.getbootstrap.com" target="top">Bootstrap</a> and it's use of the <a href="http://lesscss.org/" target="top">LESS</a> pre-processor.
+          If you prefer to use PatternFly with <a href="http://sass-lang.com/" target="top">SASS</a>, you can find that <a href="https://github.com/patternfly/patternfly-sass" target="top">here</a>.
+        </div>
+      </div>
+      <div class="col-xs-12 col-sm-12 col-md-12">
+        <p>Now create two folders - one for /less and one for /css.</p>
```

Comment:
When you say "/less" and "/css", that suggests the root file system. I think this should be something more like <install path>/less, for example?